.github/workflows/check-tidy.yml

```diff
@@ -9,7 +9,7 @@
 # licenses/APL.txt.
 
 # This workflow checks that go mod tidy command we have set for the specific
-# go version is not broken, for example `go mod tidy -go=1.21.3`. This
+# go version is not broken, for example `go mod tidy -go=1.2x.3`. This
```

Comment:
nitpick: 1.2x.3 is a bit odd, that's not really how version numbers work IMO :)

Reply:
you are right will remove the entire comment

Reply:
- [x] done